lexicons/app/bsky/actor/searchActor.json

```diff
@@ -1,6 +1,6 @@
 {
   "lexicon": 1,
-  "id": "app.bsky.actor.search",
+  "id": "app.bsky.actor.searchActor",
```

Comment:
I see this is exactly what's reflected in #597, just wanted to make sure we don't want `searchActors`, since it both searches through many actors and returns many actors.  Might compare, for example, singular `com.atproto.server.getSession` and plural `com.atproto.sync.subscribeRepos`.

Reply:
I'm game 👍 the argument i'd make against that is that generally when you're searching, even tho you receive multiple back, you are usually searching for a single actor

Reply:
That's a good point, hadn't considered that!  Seems reasonable either way :) pretty much just want to ensure the way it was is intentional.

Reply:
eh i decided to do it. i said them both out loud & liked plural better